### PR TITLE
os_test: define CFG_TA_FLOAT_SUPPORT in sub.mk

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: ta/os_test/sub.mk [hunk 1]
```diff
@@ -1,3 +1,7 @@
+ifeq ($(CFG_TA_FLOAT_SUPPORT),y)
+cppflags$(sm) += -DCFG_TA_FLOAT_SUPPORT=1
```
Comment (jenswi-linaro) [inline ta/os_test/sub.mk]: `cppflags-$(CFG_TA_FLOAT_SUPPORT) += -DCFG_TA_FLOAT_SUPPORT=1` should be sufficient.

**Comment by pascal-brand-st-dev** [comment]
Updated

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**